api/v1alpha1/instaslice_types.go

```diff
@@ -182,6 +182,12 @@ type InstasliceStatus struct {
 	// nodeResources specifies the discovered resources of the node
 	// +optional
 	NodeResources DiscoveredNodeResources `json:"nodeResources"`
+	// ObservedGeneration tracks the latest generation of the resource that has been observed and acted upon by the controller
+	// +optional
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+	// IsMetricProcessed specifies the metrics is updated
+	// +optional
+	IsMetricProcessed bool `json:"isMetricProcessed,omitempty"`
```

Comment:
The best practice is to define this as a pointer to the boolean and not the boolean

Reply:
removed IsMetricProcessed